coaster/db.py

```diff
@@ -1,5 +1,43 @@
 # -*- coding: utf-8 -*-
 
-from flask.ext.sqlalchemy import SQLAlchemy
+from __future__ import absolute_import
+from sqlalchemy import exc
+from flask.ext.sqlalchemy import SQLAlchemy as SQLAlchemyBase, SignallingSession
+
+__all__ = ['SQLAlchemy', 'db']
+
+
+class CoasterSession(SignallingSession):
+    """
+    Custom session that provides additional helper methods.
+    """
+
+    def add_and_commit(self, _instance, **filters):
```

Comment:
How about `persist`? With reference to SQLAlchemy's [Persistent](http://docs.sqlalchemy.org/en/rel_0_9/orm/session_state_management.html) state.